### PR TITLE
feat(start appeal email): email sent on valid review outcome confirmation

### PR DESCRIPTION
**Pull request by biroldif**

### CODE REVIEW

File: packages/web-app/src/lib/get-address.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    siteAddress?.town,
+    siteAddress?.county,
+    siteAddress?.postcode,
+  ].filter((n) => n);
+
+module.exports = {
+  getAddress,
+  getAddressSingleLine: (siteAddress) => getAddress(siteAddress).join(', '),
+  getAddressMultiLine: (siteAddress) => getAddress(siteAddress).join('\n'),
+};
```
Comment (shepda) [inline packages/web-app/src/lib/get-address.js:14]: It appears that these methods are only used in the unit tests? If so, I think a better location for the file would be `test/utils`.

Comment (biroldif) [reply packages/web-app/src/lib/get-address.js:13]: i removed getAddress. getAddressSingleLine and getAddressMultiLine are used in notify.js

Comment (shepda) [reply packages/web-app/src/lib/get-address.js:13]: :+1:

**Review by shepda — APPROVED** [review]
LGTM

**Pull request merged**